src/components/ReplyCard/InReplyTo/InReplyToBase.vue

```diff
@@ -62,6 +98,24 @@
       isSelectType() {
         return this.inReplyTo.type === 'application/vnd.lime.select+json'
       },
+      isImageReply() {
+        return this.inReplyTo.value.type === 'image/png'
+      },
+      isVideoReply() {
+        return this.inReplyTo.value.type === 'video/mp4'
+      },
+      isDocumentReply() {
+        return this.inReplyTo.value.type === 'application/pdf'
+      },
+      isAudioReply() {
+        return this.inReplyTo.value.type && this.inReplyTo.value.type.includes('audio')
+      },
+      isLocationReply() {
+        return this.inReplyTo.type.includes('location')
+      },
+      isUnsupportedContentReply() {
+        return this.inReplyTo.type === 'template'
```

Comment:
Should not this check be like this? If you remove `isUnsupportedContentReply` from `isAcceptableTextType`.

```suggestion
        return !this.isAcceptableTextType
```